hc/accounts/tests/test_login.py

```diff
@@ -17,15 +17,15 @@ def test_it_sends_link(self):
         r = self.client.post("/accounts/login/", form)
         assert r.status_code == 302
 
-        # ## **Assert that a user was created
+        # Assert that a user was created #
```

Comment:
I added the '''#''' at the end of the comment to mark the ones that we were tasked with as done.